cpp/src/neighbors/ball_cover/registers-ext.cuh

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
nitpick: since we're no longer header only - I'd prefer to get rid of having `-ext.cuh` and `-inl.cuh` header files.

Reply:
Do you mean just consolidate them into one file or do you mean remove the intantiations / ignored extern templates?